src/main/java/com/gmail/goosius/siegewar/command/SiegeWarCommand.java

```diff
@@ -132,4 +159,86 @@ private void parseSiegeWarHudCommand(Player player, String[] args) {
 			Messaging.sendErrorMsg(player, e.getMessage());
 		}
 	}
+
+	private void parseSiegeWarNationCommand(Player player, String[] args) {
+		if (args.length < 2) {
+			showNationHelp(player);
+			return;
+		}
+
+		if (!player.hasPermission(SiegeWarPermissionNodes.SIEGEWAR_COMMAND_SIEGEWAR_NATION.getNode(args[0]))) {
+			player.sendMessage(Translation.of("msg_err_command_disable"));
+			return;
+		}
+
+		switch (args[0]) {
+			case "paysoldiers":
+				try {
+					if (!SiegeWarSettings.getWarSiegeMilitarySalaryEnabled()) {
+						player.sendMessage(Translation.of("msg_err_command_disable"));
+						return;
+					}
+
+					if (!player.hasPermission(SiegeWarPermissionNodes.SIEGEWAR_COMMAND_SIEGEWAR_NATION_PAYSOLDIERS.getNode())) {
+						player.sendMessage(Translation.of("msg_err_command_disable"));
+						return;
+					}

```

Comment:
This has already been tested for, on line 169, and can be removed. 

Any time the .getNode(NOTEMPTY) is used on a permission node that ends with `.*` (like the `SIEGEWAR_COMMAND_SIEGEWAR_NATION("siegewar.command.siegewar.nation.*")` used on line 169,) it will replace the `.*` with the given args[0] and look for the child node instead.

Reply:
Fixed